packages/core/src/components/overflow-list/overflowList.tsx

```diff
@@ -151,7 +151,7 @@ export class OverflowList<T> extends React.Component<OverflowListProps<T>, IOver
     /** A cache containing the widths of all elements being observed to detect growing/shrinking */
     private previousWidths = new Map<Element, number>();
 
-    private spacer: Element | null = null;
+    private spacer: HTMLElement | null = null;
```

Comment:
Technically this no longer allows non-HTML usage of the overflow list, but given that it's relatively specific for HTML layout, I'm ok with this.